homeassistant/components/imap/strings.json

```diff
@@ -43,7 +43,7 @@
       "message": "Marking the the message for deletion failed with \"{error}\"."
     },
     "expunge_failed": {
-      "message": "Expungling the the message failed with \"{error}\"."
+      "message": "Expunging the the message failed with \"{error}\"."
```

Comment:
What is an expunge in the first place?

Reply:
Deleting an email does not really delete the email in the IMAP protocol, it is rather only marked for deletion. The IMAP EXPUNGE command is then used to really delete the e-mails on the server which are marked for deletion.

Reply:
A definite form of erasing. The IMAP command is called this way.